iac-extensions/cloudformation/src/main/resources/org/sonar/l10n/cloudformation/rules/cloudformation/S1135.html

```diff
@@ -0,0 +1,12 @@
+<p>FIXME: add a description</p>
+<h2>Noncompliant Code Example</h2>
+<pre>
+FIXME
+</pre>
+<h2>Compliant Solution</h2>
+<pre>
+FIXME
+</pre>
+<h2>See</h2>
+<p>FIXME: A list of links</p>
```

Comment:
@rudy-regazzoni-sonarsource sometimes it is also worth to check the rule metadata :-D

Reply:
Ah indeed ! Will pay more attention next time, thanks !